.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
-  parser: "babel-eslint",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.